projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/vdk-jupyterlab-extension/vdk.py

```diff
@@ -34,8 +34,11 @@ def run_job(path, arguments=None):
             )
             cmd = shlex.split(cmd_str)
```

Comment:
Seems that manually appending to one giant string then splitting, defeats the fix purpose:
```
cmd_str = (
                f"vdk run {path}"
                if not arguments
                else f"vdk run {path} --arguments {arguments}"
             )
cmd = shlex.split(cmd_str)
```
I think, the parameterised cmd items should be added separately to the list `cmd`.